Networking/Sources/Networking/Requests/FilterByMainIngredientAPIRequest.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+// MARK: - FilterByMainIngredientAPIResponse
+public struct FilterByMainIngredientAPIResponse: Codable {
+    let meals: [Meal]
+}
+
+// MARK: - Meal
+struct Meal: Codable {
+    let strMeal: String
+    let strMealThumb: String
+    let idMeal: String
```

Comment:
Please re-write the names with more descriptive names and use coding keys to match the endpoint values.